### DISCUSSION TITLE
Continuous Integration (CI) Guide - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#4169] [Docs]
# Continuous Integration (CI) Guide - Ultralytics YOLOv8 Docs

Learn how Ultralytics leverages Continuous Integration (CI) for maintaining high-quality code. Explore our CI tests and the status of these tests for our repositories.

https://docs.ultralytics.com/help/CI/

**Comment by glenn-jocher** [comment]
Hi everyone! This is our Docs Continuous Integration (CI) page where we've consolidated all of our CI results into one easy to read table. At a glance we can see how all systems across all main Ultralytics repositories are functioning.

**All tests should be green at all times.** If you spot any failing tests, please let us know in the relevant repo and we will investigate promptly. Thank you!
